dags.yaml

```diff
@@ -522,16 +522,16 @@ bqetl_desktop_platform:
 bqetl_internal_tooling:
   description: |
     This DAG schedules queries for populating queries related to Mozilla's
-    internal developer tooling (e.g. mozregression).
+    internal developer tooling (e.g. mozregression and Firefox-CI).
   default_args:
     depends_on_past: false
     email:
-      - wlachance@mozilla.com
+      - ahalberstadt@mozilla.com
```

Comment:
wlachance is no longer with Mozilla. Zeid Zabaneh is the new mozregession maintainer, but I'm also very involved with internal tooling so happy to take over ownership here.